app/src/main/java/com/example/sharingang/UserProfileViewModel.kt

```diff
@@ -24,7 +39,53 @@ class UserProfileViewModel @Inject constructor(
             }
         }
 
+    fun refreshListUI(viewModel: ItemsViewModel){
+        viewModelScope.launch(Dispatchers.IO){
+            if(userId.value!= null){
+                val itemList = ArrayList<Item>()
+                for(str in userRepository.get(userId.value!!)!!.wishlist){
```

Comment:
I think you don't need the "!!" for userId.value!!, it's null checked at line 44

Reply:
I think he told me even though he null checks it, android studio was complaining here without it